lib/index.js

```diff
@@ -1,7 +1,10 @@
+var angular = require('angular');
 require('dragged');
+require('splitbox');
+
+angular.module('ui.winwin', ['ui.dragged', 'ui.box']);
```

Comment:
This is still locked to version 0.0.2. that's why it's ui.box. there are some integration issues that I didn't fix yet. it'll come as a separate PR